### PR TITLE
add new term UBERON:800008 ankle/talocrural region

### LINKED ISSUE #1626: ankle/talocrural region
**Preferred term label:**
ankle/talocrural region

**Synonyms**
ankle region - broad (remove ankle region as synonym for UBERON:0004454 tarsal region)
ankle bones - narrow

**Definition (free text, please give PubMed ID)**
A region that overlaps the 3 bone elements in the ankle joint: the tibia (shinbone), the fibula (outer ankle bone), and the talus (which is the bone that connects your leg to your foot).
Source: https://www.hss.edu/condition-list_ankle-fractures.asp

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
UBERON:0004454 tarsal region

Needed for the logical def for:
HP_0041153 'Fractured ankle'

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #1626

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
should it be called 'talocrural region' then?

Is there a relation called 'overlapping in'?

**Comment by cmungall** [comment]
I prefer to avoid slashes in primary labels - commit to one schema or another

I think for now just say overlaps some femur etc. We can add directionality later

**Comment by wdahdul** [comment]
I'm not familiar with this term, so can't comment on the definition or term name - sorry!

**Comment by dosumis** [comment]
@cmungall - I think down to you to finalise review and approve is poss - given your detailed comment.  Ta

**Comment by matentzn** [comment]
What are the next steps here please?

**Comment by nicolevasilevsky** [comment]
I made the updates that @cmungall requested - this is ready for review, thanks!

**Comment by cmungall** [comment]
This doesn't address my comment from a year ago:

> Is it really a tarsal region? I don't think so, as tarsal part of foot, and foot is spatially disjoint from tib/fib

We need some documentation on mereology. For now this will have to do: https://github.com/obophenotype/uberon/wiki/Part-disjointness-Design-Pattern

It seems the use case for this term is defining HPO ankle fracture

this is low priority, there are zero associations https://hpo.jax.org/app/browse/term/HP:0041153

I think the HPO text def could be improved, and why isn't there specific subclasses? If we are serious about using HPO to classify fractures then we should have the basic subtypes of ankle fracture (each of which is easier to axiomatize using a distinct uberon bone or bone structure, such as malleolus)

But if you really want to axiomatize HPO ankle fracture, just use uberon ankle joint. I think it expresses precisely what you're trying to capture here

**Comment by nicolevasilevsky** [comment]
Thanks @cmungall! I'll use uberon ankle joint in the HPO term. i think we can close this.
